modules/33acrossIdSystem.js

```diff
@@ -69,10 +77,15 @@ export const thirthyThreeAcrossIdSubmodule = {
    * @returns {{'33acrossId':{ envelope: string}}}
```

Comment:
I think we should update this comment to reflex that we also return an extension object.